provision/acc_provision/templates/acc-provision-crd.yaml

```diff
@@ -0,0 +1,150 @@
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: accprovisioninputs.aci.ctrl
+spec:
+  group: aci.ctrl
+  names:
+    kind: AccProvisionInput
+    listKind: AccProvisionInputList
+    plural: accprovisioninputs
+    singular: accprovisioninput
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+    subresources:
+      status: {}
+    schema:
+      openAPIV3Schema:
+        description: accprovisioninput owns the lifecycle of ACI objects in the cluster
+        properties:
+          apiVersion:
+            type: string
+          kind:
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: AccProvisionInputSpec defines the desired spec for ACI Objects
+            properties:
+              acc_provision_input:
+                type: object
+                properties:
+                  aci_config:
+                    type: object
+                    properties:
+                      cluster_tenant:
+                        type: string
+                      sync_login:
+                        type: object
+                        properties:
+                          certfile:
+                            type: string
+                          keyfile:
+                            type: string
+                      client_ssl:
+                        type: boolean
+                  net_config:
```

Comment:
When comparing with the following, some fields are missing:
https://github.com/noironetworks/acc-provision/blob/master/provision/acc_provision/acc_provision.py#L161-L176
(namely, service_monitor_interval, disable_wait_for_network, duration_wait_for_network)